README.md

```diff
@@ -17,8 +17,14 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@v2-beta
+      with:
+        fetch-depth: 0
     - uses: simplabs/ember-asset-size-action@v1
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
 ```
+
+Note: as this action requires access to the "base" commit of a PR branch we need to fetch the whole repo by adding `fetch-depth: 0` to the `actions/checkout` configuration.
+
+If you would like to follow discussion on this issue you can follow: https://github.com/actions/checkout/issues/93
```

Comment:
this seems unrelated to `yarn` support, doesn't it?

Reply:
Yes but this suggested config broke 2 days ago because upstream changed something major (also why we're recommending targeting a tag instead of `@master` now).

Do you want me to split it into 2 PRs?

Reply:
> Do you want me to split it into 2 PRs?

I think that would be appropriate 👍

Reply:
@Turbo87 done: https://github.com/simplabs/ember-asset-size-action/pull/4